cli/src/pcluster/cli/commands/dcv_connect.py

```diff
@@ -152,7 +182,7 @@ class DcvConnectCommand(CliCommand):
 
     # CLI
     name = "dcv-connect"
-    help = "Permits to connect to the head node through an interactive session by using NICE DCV."
+    help = "Permits connection to the head and login nodes through an interactive session by using NICE DCV."
```

Comment:
Nitpick: You should change this to `Permits connection to the head or login nodes` instead of `and`